src/textures/mlx_xpm42.c

```diff
@@ -37,6 +37,17 @@
  * straight forward to this format however.
  */
 
+/**
+ * Parses HEX color channel e.g: "0F"
+ * 
+ * @param channel The 2 character string to parse.
+ * @return Int value of the channel.
+ */
+static uint8_t parse_hex_channel(char *channel) {
```

Comment:
Absolute heresy!

Reply:
Also remove the `Int` in the return value, I think its redundant.

Reply:
I believe I addressed this in: f5f359e, unless you meant that you want me to move this function to _mlx_utils.c_

Reply:
Just the `*` symbol must be with the type :D

Reply:
Old habits die hard: 87879ce